pkgs/by-name/de/decent-sampler/package.nix

```
@@ -0,0 +1,65 @@
+{ lib
+, stdenv
+, fetchzip
+, buildFHSEnv
+, alsa-lib
+, freetype
+, nghttp2
+, libX11
+, }:
+
+let
+  pname = "decent-sampler";
+  version = "1.9.4";
+
+  decent-sampler = stdenv.mkDerivation {
+    inherit pname version;
+
+    src = fetchzip {
+      # dropbox link: https://www.dropbox.com/sh/dwyry6xpy5uut07/AABBJ84bjTTSQWzXGG5TOQpfa\
+
+      url = "https://archive.org/download/decent-sampler-linux-static-download-mirror/Decent_Sampler-${version}-Linux-Static-x86_64.tar.gz";
+      hash = "sha256-lTp/mukCwLNyeTcBT68eqa7aD0o11Bylbd93A5VCILU=";
+    };
+
+    installPhase = ''
+      runHook preInstall
+      install -Dm755 DecentSampler $out/bin/decent-sampler
+      runHook postInstall
+    '';
```

```suggestion
    installPhase = ''
      runHook preInstall

      install -Dm755 DecentSampler $out/bin/decent-sampler

      runHook postInstall
    '';
```
small formatting thing